docs/cache/develop/api-reference/index.mdx

```diff
@@ -97,6 +118,27 @@ Get the cache value stored for the given key.
 
 <SdkExampleTabs snippetId={'API_Get'} />
 
+### GetBatch
+
+Get the cache values stored for the given keys.
+
+| Name      | Type   | Description                                   |
+| --------- | ------ | --------------------------------------------- |
+| cacheName | String | Name of the cache.                            |
+| keys       | String[] / Bytes[] | The list of keys for which to retrieve values. |
+
+<details>
+
+<summary>Method response object</summary>
+
+* Success
+- `values()`: Record<string, string>
```

Comment:
```suggestion
- `values()`: Record&lt;string, string&gt;
```
It looks like there may be an issue here with the `<` and `>` characters in what is technically an XML snippet.